### PR TITLE
Default x y axis

### PR DESCRIPTION
**Pull request by cristiingineru**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ module.exports = function (config) {
         },
 
         eslint: {
-            stopOnError: true,
+            stopOnError: false,
```
Comment (cipix2000) [inline karma.conf.js]: ???

File: spec/cursors.Test.js [hunk 2]
```diff
@@ -457,15 +457,16 @@ describe('Flot cursors', function () {
...
+                        snapToPlot: 0
                     }
                 ]
             });
 
             var spy = spyOnFillText();
             jasmine.clock().tick(20);
 
-            expect(spy).toHaveBeenCalledWith('1.000, 1.1500', jasmine.any(Number), jasmine.any(Number));
+            expect(spy).toHaveBeenCalledWith('1.000, 1.1000', jasmine.any(Number), jasmine.any(Number));
```
Comment (cipix2000) [inline spec/cursors.Test.js]: this doesn't make much sense

File: spec/cursors.Test.js [hunk 3]
```diff
@@ -485,7 +486,7 @@ describe('Flot cursors', function () {
             var spy = spyOnFillText();
             jasmine.clock().tick(20);
 
-            expect(spy).not.toHaveBeenCalledWith('1.0000, 1.15000', jasmine.any(Number), jasmine.any(Number));
+            expect(spy).not.toHaveBeenCalledWith('1.0000, 1.1500', jasmine.any(Number), jasmine.any(Number));
```
Comment (cipix2000) [inline spec/cursors.Test.js]: why isn't showValues true by default ?

File: spec/cursors_snapping.Test.js [hunk 4]
```diff
@@ -152,6 +154,96 @@ describe("Cursors snapping", function () {
         expect(cursor.y).toBe(pos.top);
     });
 
+    it('should be able to limit the snap to range to the plot area', function () {
```
Comment (cipix2000) [inline spec/cursors_snapping.Test.js]: 'should be able to limit the snap to range **of** the plot area'

File: spec/cursors.Test.js (hunk 2 above)
Comment (cristiingineru) [inline spec/cursors.Test.js]: ```showValuesRelativeToSeries: 0``` used to be equivalent to ```snapToPlot: 0, showValues: true``` and the test data is ```[[0, 1], [1, 1.1], [2, 1.2]]``` so the cursor is expected to snap to [1, 1.1]. It was an issue that auto-magically got fixed.

File: spec/cursors.Test.js (hunk 3 above)
Comment (cristiingineru) [inline spec/cursors.Test.js]: Yes, it's true by default.

File: spec/cursors.Test.js [hunk 5]
```diff
@@ -485,7 +485,7 @@ describe('Flot cursors', function () {
...
```
Comment (cipix2000) [inline spec/cursors.Test.js]: expect(spy).not.toHaveBeenCalled()

also specify showValues: false to be more clear what the tests is doing

**Pull request merged**